src/main/resources/static/browserStorage.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Browser storages</title>
```

Comment:
common name for this technology is `Web Storage`: https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API:

```suggestion
  <title>Web Storage</title>
```

Reply:
also please update HTML file name in the same way

Reply:
Both the title and the file name have been updated.